pixart.js

```diff
@@ -1 +1,18 @@
+$(document).ready(function() {
+  $('button').click(function(evt) {
+    evt.preventDefault()
+    console.log("Clicked")
+  });
+});
+$('#Set Color').bind('keypress', function(e) {
```

Comment:
Wow, I didn't expect anyone to use `.bind`. I would recommend instead using `.on("keypress"`:

```js
$("#set_color").on("keypress", function(e){
```